docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+
+services:
+  todo_app:
+    build:
+      context: .
+      target: development
+    ports:
+      - 5000:80
+    volumes:
+      - type: bind
+        source: todo_app
+        target: /app/todo_app/todo_app
+    env_file:
+      - .env
+
+  todo_app_unit_test_runner:
+    build:
+      context: .
+      target: test
+    volumes:
+      - type: bind
+        source: todo_app
+        target: /app/todo_app/todo_app
+    env_file:
+      - .env.test
+    command: tests
+
+  todo_app_e2e_test_runner:
+    build:
+      context: .
+      target: test
+    ports:
+      - 5100:80
```

Comment:
Minor, but I don't think this container really needs to expose any ports?

Reply:
i believe it would as this container will communicate with the trello API

Reply:
Exposing ports isn't necessary for external comms (or rather, fundamentally it is, but we don't need to do it explicitly!) - since Trello is only responding to our direct requests that should be sufficient